l10n/templates/strings.properties

```diff
@@ -2,8 +2,13 @@ newtab_page_title=New Tab
 default_label_loading=Loading…
 
 header_top_sites=Top Sites
-header_highlights=Highlights
 header_stories=Top Stories
+header_visit_again=Visit Again
+header_bookmarks=Recent Bookmarks
+# LOCALIZATION NOTE(header_bookmarks_placeholder): This message is
+# meant to inform that section contains no information because
+# the user hasn't added any bookmarks.
+header_bookmarks_placeholder=You don't have any bookmarks yet.
```

Comment:
Nit: you should use a proper apostrophe instead of a straight quote (looks like there's another one in welcome_body). This would fail tests if landed in mozilla-central